src/main/java/edu/gatech/cs2340/trydent/sample/tictactoe/TitleScreenController.java

```diff
@@ -0,0 +1,155 @@
+                    handleKeyboard();
+                } else if(started3) {
+                    handleMouse();
+                }
+            }
+        };
+    }
+
+    private void handleKeyboard() {
+        Rectangle target;
+        if(Keyboard.isKeyDownOnce(KeyCode.Q)){
+            target = tiles.get("0_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.A)){
+            target = tiles.get("0_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.Z)){
+            target = tiles.get("0_2");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.W)){
+            target = tiles.get("1_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.S)){
+            target = tiles.get("1_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.X)){
+            target = tiles.get("1_2");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.E)){
+            target = tiles.get("2_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.D)){
+            target = tiles.get("2_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.C)){
+            target = tiles.get("2_2");
+        } else {
+            target = null;
+        }
+        if(target != null && !used.contains(target)) {
+            if(player) {
+                target.setFill(Color.RED);
+            } else {
+                target.setFill(Color.BLUE);
+            }
+            player = !player;
+            used.add(target);
+        }
+    }
+
+    private void handleMouse() {
+        if(Mouse.isMouseDownOnce(MouseButton.PRIMARY)){
+            for(int i = 0; i < 3; i++){
+                for(int j = 0; j < 3; j++){
+                    Rectangle target = tiles.get(i + "_" + j);
+                    if(target.contains(target.sceneToLocal(Mouse.getMouseX(), Mouse.getMouseY()))
+                            && !used.contains(target)) {
+                        if(player) {
+                            target.setFill(Color.RED);
+                        } else {
+                            target.setFill(Color.BLUE);
+                        }
+                        player = !player;
+                        used.add(target);
+                    }
+                }
+            }
+        }
+    }
+
+    @FXML
+    private void toGameScreen(ActionEvent event) {
+        OneTimeEvent.simplified(() -> {
+                if(started2 || started3) return;
+                Log.debug("Switching to game screen.");
+                TrydentEngine.setForeground(getClass().getResource("TicTacToeGameScreen.fxml"));
+            }
+        );
+    }
+
+    private void initEventScreen() {
+        tiles = new HashMap<>();
+        used = new HashSet<>();
+        for(int i = 0; i < 3; i++){
+            for(int j = 0; j < 3; j++){
+                String name = i + "_" + j;
+                Rectangle rect = new Rectangle(50, 50, Color.BLACK);
+                tiles.put(name, rect);
+                (new GameObject(name, rect)).setPosition(new Position(i * 60, j * 60));
```

Comment:
Parenthesis aren't needed around the new object creation, but I suppose you might want them for clarity.